### ISSUE TITLE
[Bug] Velocity Automation works only for patterns with length = 192

### ISSUE THREAD

**Issue by oddtime** [hydrogen-music/hydrogen#1240]
Way to reproduce: write a velocity automation ramp with value from 0 to 1 during one pattern of length = 8/8 (192*2 ticks), previously filled with notes.

I hear like if there were TWO ramps from 0 to 1

I think that the problem is here in `Hydrogen.cpp`:
```
		float velocity_adjustment = 1.0f;
		if ( pSong->getMode() == Song::SONG_MODE ) {
			float fPos = m_nSongPos + (pNote->get_position()%192) / 192.f;
			velocity_adjustment = vp->get_value(fPos);
		}
```

where that modulo `%192 `resets the positions periodically, instead that should depend on the biggest pattern size at that moment in the sequence.
If somebody confirms the bug I can work on it.

**Comment by oddtime** [comment]
I have opened pull req #1241

**Comment by theGreatWhiteShark** [comment]
Closed with #1241
